modules/admin-quota-usage.adoc

```diff
@@ -0,0 +1,366 @@
+
+To configure an object count quota for a resource, run the following command:
+
+[source,terminal]
+----
+$ oc create quota <name> --hard=count/<resource>.<group>=<quota>,count/<resource>.<group>=<quota>
+----
+
+For example:
+
+[source,terminal]
+----
+$ oc create quota test --hard=count/deployments.extensions=2,count/replicasets.extensions=4,count/pods=3,count/secrets=4
+resourcequota "test" created
+
+$ oc describe quota test
+Name:                         test
+Namespace:                    quota
+Resource                      Used  Hard
+--------                      ----  ----
+count/deployments.extensions  0     2
+count/pods                    0     3
+count/replicasets.extensions  0     4
+count/secrets                 0     4
+----
+
+This example limits the listed resources to the hard limit in each project in the cluster.
+
+== Viewing a quota
+
+You can view usage statistics related to any hard limits defined in a project's quota by navigating in the web console to the project's `Quota` page.
+
+You can also use the CLI to view quota details:
+
+. First, get the list of quotas defined in the project. For example, for a project called `demoproject`:
++
+
+[source,terminal]
+----
+$ oc get quota -n demoproject
+NAME                AGE
+besteffort          11m
+compute-resources   2m
+core-object-counts  29m
+----
+
+
+. Then, describe the quota you are interested in, for example the `core-object-counts` quota:
++
+
+[source,terminal]
+----
+$ oc describe quota core-object-counts -n demoproject
+Name:			core-object-counts
+Namespace:		demoproject
+Resource		Used	Hard
+--------		----	----
+configmaps		3	10
+persistentvolumeclaims	0	4
+replicationcontrollers	3	20
+secrets			9	10
+services		2	10
+----
+
+ifdef::openshift-origin,openshift-enterprise[]
+
+== Configuring quota synchronization period
+
+When a set of resources are deleted, the synchronization time frame of resources is determined by the `*resource-quota-sync-period*` setting in the
+*_/etc/origin/master/master-config.yaml_* file.
+
+Before quota usage is restored, a user can encounter problems when attempting to reuse the resources. You can change the `*resource-quota-sync-period*` setting to have the set of resources regenerate in the needed amount of time (in seconds) for the resources to be once again available:
+
+
+[source,yaml]
+----
+kubernetesMasterConfig:
+  apiLevels:
+  - v1beta3
+  - v1
+  apiServerArguments: null
+  controllerArguments:
+    resource-quota-sync-period:
+      - "10s"
+----
+
+
+After making any changes, restart the controller services to apply them.
+
+[source,terminal]
+----
+$ master-restart api
+$ master-restart controllers
+----
+
+Adjusting the regeneration time can be helpful for creating resources and determining resource usage when automation is used.
+
+[NOTE]
+====
+The `resource-quota-sync-period` setting is designed to balance system performance. Reducing the sync period can result in a heavy load on the controller.
```

Comment:
@snarayan-redhat I went a little further, because I don't think "helps" is strong enough. I simply deleted "is designed to" and added an "s" to "balance". I believe that makes the text more definitive, more effective.